truss-chains/examples/text_to_num/main.py

```diff
@@ -174,7 +174,7 @@ async def run(self, length: int, num_partitions: int) -> tuple[int, str, int]:
 
     import asyncio
 
-    class FakeMistralLLM(chains.ChainletBase):
+    class FakeMistralLLM:
```

Comment:
why is this no longer a chainlet?

Reply:
Because it's a "fake". To keep it a Chainlet, it would require adding `remote_config` (became mandatory), so instead of doing that I just dropped the baseclass.

From a typing perspective, what we actually care about is this:

Each "actual" user-defined chainlet implies a `Protocol`, specifically the signature of the `run` method (e.g. `MistralP`, [should have the `__init__` removed, because that doesn't matter]).
Ideally we would change all type annotations where chainlets are injected as dependencies to the protocol (structural typing) instead of the class (nominal typing).

However, I don't see users writing an extra protocol for each chainlet, so we can just scratch this aspiration.

But still, we should be able to substitute any instance for a chainlet that implements the corresponding protocol (this is basically what is going on with the stubs for remote deployment too, the *aren't* chainlet subclasses [but `StubBase` subclasses]).